src/Service/CprService.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Drupal\os2forms_cpr_lookup\Service;
+
+use Drupal\os2forms_cpr_lookup\CPR\CprServiceResult;
+use Http\Factory\Guzzle\RequestFactory;
+use ItkDev\AzureKeyVault\Authorisation\VaultToken;
+use Http\Adapter\Guzzle6\Client as GuzzleAdapter;
+use ItkDev\AzureKeyVault\KeyVault\VaultSecret;
+use ItkDev\Serviceplatformen\Certificate\AzureKeyVaultCertificateLocator;
+use ItkDev\Serviceplatformen\Request\InvocationContextRequestGenerator;
+use ItkDev\Serviceplatformen\Service\PersonBaseDataExtendedService;
+
+class CprService
+{
+  private $personBaseDataExtendedService;
+
+  public function __construct()
+  {
+    $guzzleClient = \Drupal::httpClient();
+    $config = \Drupal::config('os2forms_cpr_lookup');
```

Comment:
We should use dependency injection for this.

Reply:
Fixed in https://github.com/itk-dev/os2forms_cpr_lookup/pull/7